src/lyapunov_spectrum.jl

```diff
@@ -154,14 +132,16 @@ for a given particle in a billiard table.
 """
 function lyapunovspectrum!(p::AbstractParticle{T}, bt::Billiard{T}, t::T) where {T<:AbstractFloat}
 
```

Comment:
Can you add a conversion that makes `t` of type `T`, so that this restriction is not present in the call signature?